site/content/en/docs/administration/advanced/analytics.md

```diff
@@ -1,24 +1,389 @@
+| **user_id**            | User ID.                                                                                                                                                                                                                           |
+| **user_name**          | User nickname.                                                                                                                                                                                                                     |
+| **project_id**         | Id of the project. Might be empty.                                                                                                                                                                                                 |
+| **task_id**            | Task ID. Might be empty.                                                                                                                                                                                                           |
+| **job_id**             | Job ID. Might be empty.                                                                                                                                                                                                            |
+| **error**              | Error description                                                                                                                                                                                                                  |
+| **stack**              | Error description                                                                                                                                                                                                                  |
+| **payload**            | Error description                                                                                                                                                                                                                  |
+| **stack**              | Stack trace, which is a report of the active stack frames at a certain point in time during the execution. This information is typically used for debugging purposes to locate where an issue occurred.                            |
+| **payload**            | JSON that describes the entire object, which contains several properties. This data in the payload is related to an event that was created as a result of a failed API request. The payload contains information about this event. |
+
+<!--lint enable maximum-line-length-->
+
+Every column name can be used like a filter.
+
+If you want to inspect the value, hover over it and click
+on the eye icon.
+
+### Dashboards update
+
+You can adjust the dashboards. To do this, click on the
```

Comment:
It's good idea to add instruction how to adjust dashboards, let's disscuss offline.